packages/contracts/src/contracts/current/protocol/Exchange/MixinSignatureValidator.sol

```diff
@@ -28,12 +28,13 @@ contract MixinSignatureValidator is MSignatureValidator {
     enum SignatureType {
         Illegal, // Default value
         Invalid,
-        Caller,
-        Ecrecover,
         EIP712,
-        Trezor,
+        Ecrecover,
+        TxOrigin,
+        Caller,
         Contract,
-        PreSigned
+        PreSigned,
+        Trezor
```

Comment:
Why were these re-ordered?

Reply:
I ordered them by how frequently I think they will be used.